tests/functional/builtins/codegen/test_create_functions.py

```diff
@@ -686,10 +682,10 @@ def test(target: address) -> address:
     """
 
     c = get_contract(complex_salt)
-    bytecode = w3.eth.get_code(c.address)
-    c.test(c.address, transact={})
-    test1 = c.created_address()
-    assert w3.eth.get_code(test1) == bytecode
+    bytecode = env.get_code(c.address)
+    c.test(c.address)
+    test1 = c.address
+    assert bytecode and env.get_code(test1) == bytecode
```

Comment:
nit: the `assert bytecode` is a sanity check right? can we put it on its own line then since it is asserting something else

Reply:
If get_code would always return None, the test would pass. So I added this check. I'll move it